### ISSUE TITLE
Add test data for "User Security Dashboard" dashboard "References to temporary files" panel

### ISSUE THREAD

**Issue by rishagg01** [cisagov/LME#362]
Use API Post method to add test data for "User Security Dashboard" dashboard "References to temporary files" panel

**Comment by rishagg01** [comment]
Blocked by issue #422

**Comment by rishagg01** [comment]
No longer needed
